cdc/puller/frontier/frontier_bench_test.go

```diff
@@ -49,7 +49,7 @@ func BenchmarkSpanFrontier(b *testing.B) {
 				spans = append(spans, span)
 			}
 
-			f := NewFrontier(spans...)
+			f := NewFrontier(0, spans...)
```

Comment:
Do we have a benchmark result after this PR.

Reply:
updated at this issue